src/librustdoc/html/render/cache.rs

```diff
@@ -184,8 +184,8 @@ crate fn build_index<'tcx>(krate: &clean::Crate, cache: &mut Cache, tcx: TyCtxt<
         })
         .expect("failed serde conversion")
         // All these `replace` calls are because we have to go through JS string for JSON content.
-        .replace(r"\", r"\\")
-        .replace("'", r"\'")
+        .replace('\\', r"\\")
+        .replace('\'', r"\'")
```

Comment:
Hmm, I think was more clear before - can you change both of these to raw string?